src/components/Calculator.css

```diff
@@ -7,14 +7,16 @@
 
 body {
   background-color: aquamarine;
-  display: flex;
-  justify-content: center;
-  align-items: center;
+}
+
+h1 {
+  font-size: 50px;
 }
 
 .container {
-  top: 50%;
-  left: 50%;
+  top: 60%;
+  left: 70%;
+  width: 50%;
```

Comment:
[Optional]
- [ ] Using absolute positioning for your HTML elements is considered a bad practice.  Absolute positioning overrides the default flow layout of browsers and so will reduce accessibility for many users. The page will look different on different screen resolutions. For illustration, I took screenshots on my desktop resolution and laptop resolution and attached them so you can compare.

| Laptop | Desktop |
| --------- | ----------- |
| ![Screenshot 2021-12-03 at 02-21-28 React App](https://user-images.githubusercontent.com/1642014/144518555-2e037d0d-a2e4-40c7-b82a-ae4774c67959.png) | ![Screenshot 2021-12-03 at 02-18-11 React App](https://user-images.githubusercontent.com/1642014/144518520-bd08c452-0519-4f46-8949-e1d9fc000c98.png) |

The best practice for positioning element is considered as using margin, padding, or using positioning features of grid and flex